binding.gyp

```diff
@@ -40,7 +40,8 @@
             'include_dirs': [
                 '/home/wace/dev/nvm/src/node-v0.10.5/src',
                 '/home/wace/dev/nvm/src/node-v0.10.5/deps/v8/include',
-                '/home/wace/dev/nvm/src/node-v0.10.5/deps/uv/include'
+                '/home/wace/dev/nvm/src/node-v0.10.5/deps/uv/include',
+		"<!(node -e \"require('nan')\")"
```

Comment:
While the indentation seems to be a bit of a mess in the project, binding.gyp seems to be using spaces in general. The new line is indented with tabs though.

.. also I had no idea I still had my old include dirs here which probably shouldn't be there!

Any chance you could change the indent for the `require('nan')` to use spaces and while doing so remove the old `/home/wace` includes?